.github/workflows/build-tarball.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build-tarball:
     env:
       PYTHON_VERSION: 3.8
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
Changed this based on feedback in the PR review.

As we use OSX for building the tarball in production we might want to have a nightly job in travis to ensure that the pipeline we will be using in production is tested in some capacity.